app/views/users/_search.html.slim

```diff
@@ -0,0 +1,4 @@
+= form_tag users_path, method: :get, id: "search-form" do |f|
+  = text_field_tag :search, params[:search], placeholder: "Search by name"
+  = submit_tag "Go", class: "btn btn-default"
+  = link_to_if(params[:search], "Clear search results", users_path)
```

Comment:
`params[:search].present?` would be better in case the param got carried over as `…?search=`, i.e. w/o a value that could be interpreted as an empty string. An empty string is still a truthy value in Ruby.